dev/doc/README.md

```diff
@@ -28,6 +43,9 @@ $ make -f Makefile.dune world
     # to build some test project
 ```
 
+When running the commands above, you may set `DUNEOPT=--display=short`
+for a more verbose build.
+
```

Comment:
Or alternatively update `.config/dune/` to set your display pref.

Reply:
I didn't find anywhere in the Dune documentation documenting this. The only place that I know of that talks about this file is https://dune.readthedocs.io/en/stable/caching.html#configuration

Reply:
Seems to be only in `man dune-config`, indeed the last rewrite of the Dune doc was not very convincing IMO.

Reply:
Removed the Coq section, forgot to add a section for the config file, etc... I think that section was there before.

Reply:
OK thanks for the tip about `man dune-config`.